<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplicationSettingsDialog</class>
 <widget class="QDialog" name="ApplicationSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Custom application settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="windowNameLabel">
       <property name="text">
        <string>Window name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="windowNameEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="windowSelectButton">
       <property name="toolTip">
        <string>Click to select the application window</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="dontProcessCheckBox">
     <property name="text">
      <string>Don't process the input for this application</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="storeLayoutCheckBox">
     <property name="text">
      <string>Store layout for the whole application, not window</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="layoutSwitchingGroup">
     <property name="title">
      <string>Layout switching</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="forceAutomaticalRadioButton">
        <property name="text">
         <string>Force automatical layout switching</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="forceManualRadioButton">
        <property name="text">
         <string>Force manual layout switching</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>windowNameEdit</tabstop>
  <tabstop>windowSelectButton</tabstop>
  <tabstop>dontProcessCheckBox</tabstop>
  <tabstop>storeLayoutCheckBox</tabstop>
  <tabstop>layoutSwitchingGroup</tabstop>
  <tabstop>forceAutomaticalRadioButton</tabstop>
  <tabstop>forceManualRadioButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ApplicationSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ApplicationSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>layoutSwitchingGroup</sender>
   <signal>toggled(bool)</signal>
   <receiver>forceAutomaticalRadioButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>layoutSwitchingGroup</sender>
   <signal>toggled(bool)</signal>
   <receiver>forceManualRadioButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
